src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/AesCornerTests.cs

```diff
@@ -324,5 +324,115 @@ public static void ZeroLengthTransformPKCS7()
                 }
             }
         }
+
+#if netstandard17
```

Comment:
Should we move this to a different file and make it a partial class? That is the norm we have been following so far.